src/interaction/command/copers.ts

```diff
@@ -0,0 +1,52 @@
+import SlashCommand, {
+  SlashCommandContext,
+} from "../../api/interaction/slashcommand";
+import { MessageEmbed, TextChannel } from "discord.js";
+import { settings } from "../../settings";
+import { APIEmbedField } from "discord-api-types";
+
+export default class CopersCommand extends SlashCommand {
+  public constructor() {
+    super({
+      name: "copers",
+      description: "Find out who's coping.",
+    });
+  }
+
+  protected buildSlashCommand() {
+    this.slashCommand;
+  }
+
+  // Interaction Handled !
+  public async handleInteraction({
+    bot,
+    interaction,
+  }: SlashCommandContext): Promise<void> {
+    const { member, guild } = interaction;
+
+    let embed = new MessageEmbed({
+      title: "Fetching copers...",
+    });
+    await interaction.reply({ embeds: [embed], ephemeral: true });
```

Comment:
Do we need this? Why not just send the result embed as the reply, non-ephemeral? Will save channel resolution logic